python/python/bystro/api/auth.py

```diff
@@ -89,7 +89,7 @@ class UserProfile(Struct, rename="camel"):
     email: str
     accounts: list[str]
     role: str
-    last_login: datetime.datetime
+    lastLogin: datetime.datetime
```

Comment:
Again not needed

Reply:
One thing you can do is make last_login optional. It doesn’t exist immediately after signup; a better solution would be fix that in bystro-web however.